frontend/src/components/editable-cell.tsx

```diff
@@ -81,6 +83,12 @@ export function EditableCell<T extends HasId, Field extends keyof T = keyof T>({
     }
 
     const formattedValue = isDate ? ("" + value).slice(0, 10) : value || "";
+    let displayedValue = isDate ? formatDate("" + value) : value;
+
+    if (isMoney) {
+        displayedValue = value != null ? formatMoney("" + value) : "";
+        type = "number";
```

Comment:
I don't like overriding input params. You should save the value in a new variable, e.g. `inputType`.